src/pages/ClaimsPage/queries.ts

```diff
@@ -80,12 +82,20 @@ export const getClaims = async (
     );
 
     const unwrappedAmount = getUnwrappedAmount(claim.amount, pricePerShare);
+    const unwrappedNewAmount = getUnwrappedAmount(
```

Comment:
the pricePerShare will increase overtime when the vault profits. So the realtime unwrapped amount will also be constantly increasing, will user be confused if this number is shown ?